app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/home/HomeScreen.kt

```diff
@@ -207,28 +248,37 @@ fun BottomView(
         )
         Spacer(modifier = Modifier.height(26.dp))
         SurveyText(
-            text = surveyResponse.attributes?.title.orEmpty(),
+            text = survey.title,
             fontSize = 28.sp,
             fontWeight = FontWeight.Bold,
             modifier = Modifier.padding(start = 20.dp, end = 20.dp)
```

Comment:
`padding()` has a `horizontal` argument 😉 
```suggestion
            modifier = Modifier.padding(horizontal = 20.dp)
```